lib/common/validate.go

```diff
@@ -36,7 +35,7 @@ func Validate(current interface{}) error {
 }
 
 // Common validation functions (accessed using the validate field tag)
-
+// [smc] More than one init() func?
```

Comment:
That's allowed - but yeah not really necessary ... I was originally hitting some issues with init() ordering that I never really got to the bottom of but got it working by init() in a different way... again another thing for me to revisit!